src/components/calcite-combobox/calcite-combobox.e2e.ts

```diff
@@ -311,11 +311,11 @@ describe("calcite-combobox", () => {
 
       await input.press("Enter");
       expect(await item1.getProperty("selected")).toBe(true);
-      expect(eventSpy).toHaveReceivedEventTimes(1);
+      expect(eventSpy).toHaveReceivedEventTimes(2);
 
       await input.press("Enter");
       expect(await item1.getProperty("selected")).toBe(false);
-      expect(eventSpy).toHaveReceivedEventTimes(2);
+      expect(eventSpy).toHaveReceivedEventTimes(4);
```

Comment:
The lookup event should only occur once on `Enter`, shouldn't it?

Reply:
Well, if there's already one selected then that one becoming unselected also fires an event.

Reply:
Technically, yes, but should it? It's one operation firing 2 change events. I think the pick/value list tackled something similar some time ago by debouncing the emitter.

Reply:
Gotcha. yeah i'll update that 👍